### DISCUSSION TITLE
There is no way to put a tenant configuration through the helm chart

### DISCUSSION THREAD

**Discussion by kdibragimov** [grafana/mimir#5119] [Help and support] [answered]
We are using the "mimir-distributed" helm chart and multitenancy is enabled. We need to use the right config for a certain tenant. With default tenant = system, I get that no config is found and fallback is used. This can be done via mimirtool (https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/alertmanager/):
mimirtool alertmanager load <ALERTMANAGER CONFIGURATION FILE>  \ --address=<ALERTMANAGER URL> --id=<TENANT ID>
But I still want it to be done automatically and through code.

**Comment by dimitarvdimitrov** [comment, marked as answer]
currently there is no way to achieve this via the helm chart. So this is more of a feature request. I see you already opened an issue for it https://github.com/grafana/mimir/issues/5101. Is it ok to close this discussion?

**Reply by kdibragimov** [reply]
yes, thanks for the answer
